dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -634,24 +669,30 @@ pub fn invoke_command(executable: &str, args: Option<Vec<String>>, input: Option
     Ok((exit_code, stdout, stderr))
 }
 
-fn replace_token(args: &mut Option<Vec<String>>, token: &str, value: &str) -> Result<(), DscError> {
+fn process_args(args: &Option<Vec<ArgKind>>, value: &str) -> Option<Vec<String>> {
     let Some(arg_values) = args else {
-        return Err(DscError::Operation("No args to replace".to_string()));
+        debug!("No args to process");
```

Comment:
Kind of too generic message.

Reply:
Wanted a debug message that may be useful in the future to know that there weren't any args